CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
with this `var target = options.expressApp || options.router;` change
it would take expressApp to attach to when both are specified
and I am working on doing a warning instead of throw an exception